src/components/pages/NoMatch/NoMatch.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, Fragment } from 'react';
+import noMatchImage from '../../../assets/images/undraw_navigation_lytx.svg';
+import './NoMatch.scss';
+
+/**
+ * @class
+ * @description 404 Class
+ */
+export default class NoMatch extends Component {
+  /**
+   * @description React lifecycle method that is invoked
+   *  when the componet mounts
+   * @returns {void} No return
+   */
+  componentDidMount() {
+    document.body.classList.add('fixed-body');
```

Comment:
@theghostyced  any particular reason why this has to be done here as opposed to setting the class directly on `main` tag

Reply:
Yeah @efalayi. So even if this is done in the `main div` and not `body`, the `body` will still be scrollable when a component overflows.

Reply:
@theghostyced  does the component have to be mounted before the class is added?

Reply:
Yes @efalayi . I want the class to take effect once the component has mounted. Or is there another that this could have been implemented?